CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Hammer Beta-2
+## Hammer RC-1
```

Comment:
This is missing the date.

Reply:
Will release date in on later PR. RC1 not released yet.

Reply:
Ah okay, spoke with @simaishi and I think I understand why this is the case, but we should call that out in any guidelines we build around the changelog because this looks wrong at first glance.